src/lib/holocene/api-pagination.svelte

```diff
@@ -226,7 +226,7 @@
       <Input
         icon="search"
         id="api-pagination-search-input"
-        class="grow"
+        class="w-1/2 grow"
```

Comment:
I would make this `w-full sm:w-1/2` so on xs screens the input takes up the full width

Reply:
Will do, thank you!